docs/colors.md

```diff
@@ -0,0 +1,53 @@
+
+# CORP Colors
```

Comment:
weird title ngl, can you change it to something like "CORP Engine - Colors"?

Reply:
absolutely no, I'll make it even weirder - what about "corplors"?

Reply:
jk, fixed